### ISSUE TITLE
Add ability to generate All Access tokens from the CLI

### ISSUE THREAD

**Issue by sanderson** [influxdata/influx-cli#66]
__Proposal:__
Add the ability to create All Access tokens from the `influx` CLI.

__Current behavior:__
Currently to grant all permissions on a token, you have to pass all the permission flags when creating the token. But even still, these tokens appear to behave differently than an All Access token created through the InfluxDB UI (you can't export a template using one of these CLI-generated tokens).

__Desired behavior:__
A single flag on `influx auth create` that generates an All Access token:

```sh
influx auth create --all-access
```

__Alternatives considered:__
- Passing all permissions when creating a token through the CLI.

__Use case:__
Ease of use when managing tokens from the command line.

**Comment by umrikar** [comment]
Which release of influxdb will have this feature

**Comment by wbaker85** [comment]
Most of the interesting stuff that happens in the UI when creating an all-access token is in this file: https://github.com/influxdata/ui/blob/master/src/authorizations/utils/permissions.ts

There is a hard-coded list of resource types to grant permissions to in the UI code, and when making a request to create an all-access token that list is used to generate a `POST` request with a body that looks something like this, containing all of the resource types listed:

```
{
   "orgID":"<org id>",
   "description":"<token name>",
   "permissions":[
      {
         "action":"read",
         "resource":{
            "type":"authorizations",
            "orgID":"<org id>"
         }
      },
      {
         "action":"write",
         "resource":{
            "type":"authorizations",
            "orgID":"<org id>"
         }
      },
      {
         "action":"read",
         "resource":{
            "type":"buckets",
            "orgID":"<org id>"
         }
      },
      {
         "action":"write",
         "resource":{
            "type":"buckets",
            "orgID":"<org id>"
         }
      },
      {
         "action":"read",
         "resource":{
            "type":"checks",
            "orgID":"<org id>"
         }
      },
      {
         "action":"write",
         "resource":{
            "type":"checks",
            "orgID":"<org id>"
         }
      },
      {
         "action":"read",
         "resource":{
            "type":"dashboards",
            "orgID":"<org id>"
         }
      },
      {
         "action":"write",
         "resource":{
            "type":"dashboards",
            "orgID":"<org id>"
         }
      },
    ... etc ...
   ]
}
```

The most obvious way to implement similar behavior in the CLI would be to have a hard-coded list in the CLI code as well to create the request. A more elegant (but less backward compatible) strategy would be to add something server-side for creating an all-access token. I could see there being an endpoint like `api/v2/authorizations/allAccess` that you could `POST` to with a minimal amount of information like the `orgID` and token description that would internally generate a token for all resources on the server, for example.

**Comment by wbaker85** [comment]
Relevant: https://github.com/influxdata/influxdb/pull/22135

**Comment by danxmoran** [comment]
Closed by #285
